app/assets/stylesheets/content/_tables.sass

```diff
@@ -60,11 +60,21 @@ table
 
   &.workflow-table
     margin-bottom: 16px
+    border-collapse: separate
+    overflow-x: hidden
     tbody
       tr:nth-of-type(2)
-        border-top: 1px solid #ddd
+        td
+          border-top: 1px solid #ddd
+
+      td
+        border-bottom: 1px solid #ddd
```

Comment:
Wouldn't any of our many grey-tones be good enough, too?

````Sass
$gray:                    #EAEAEA
$gray-dark:               #878787
$gray-light:              #F8F8F8

$light-gray: #ccc
````

(notice the well known difference between light gray and gray light...)